doc/source/guide/acquiring_data/vso.rst

```diff
@@ -57,7 +57,7 @@ response objects, each one of which is a record found by the VSO. You can find h
 records were found by typing
 
     >>> len(qr)
-    122
+    3549
```

Comment:
There's something wrong in that result... SoHO/EIT doesn't get that much data in one day.